test/sol5/kyberNetwork.js

```diff
@@ -384,10 +386,13 @@ contract('KyberNetwork', function(accounts) {
             rateHelper = await RateHelper.new(admin);
             await rateHelper.setContracts(matchingEngine.address, DAO.address, {from: admin});
 
+            // init gas helper
+            gasHelperAdd = await MockGasHelper.new(platformWallet);
+
             //setup network
             await network.addOperator(operator, {from: admin});
             await network.addKyberProxy(networkProxy, {from: admin});
-            await network.setContracts(feeHandler.address, matchingEngine.address, zeroAddress, {from: admin});
+            await network.setContracts(feeHandler.address, matchingEngine.address, gasHelperAdd.address, {from: admin});
```

Comment:
if this is the same network used in other tests.
better remove this  gas helper after the test.

Reply:
This should be okay since this change is only used within a describe. The other tests initialize another network.